src/v2/Apps/_About2/AboutApp.tsx

```diff
@@ -282,15 +282,15 @@ const AppDownloadComponent: React.FC = () => {
             height="100%"
             srcSet={image.srcSet}
             lazyLoad
-            alt=""
+            alt="painting of two iphones displaying the artsy mobile app"
```

Comment:
This isn't a painting of iPhones but rather it is two iPhones sitting on top of a painting.

However I'd also consider this a 'decorative image': https://webaim.org/techniques/alttext/#decorative in which case the blank alt is desireable.

It also looks like we'll have to replace this image: it's low quality and the credit is cut off (the credit shouldn't even be in the image IMO, but rather located below in actual text).

Reply:
I pulled that image directly from the comp, and the credit line is part of that image?

Reply:
Yeah, the comps frequently (mostly, really) don't have usable image assets — I'd ask design to provide an asset that isn't so low quality. Since the image displays at max 910x652, we'd need a minimum resolution of 1820x1304. And really the credit just looks more like a mistake to my eye than something intentional so I'd push back on that. We shouldn't be locking text inside of images anyway (if it _was_ we'd use that text verbatim in the `alt`).

Reply:
would you say the same about the other image in the section below in the `collecting` section?? I believe it was pulled from the figma comp as well.

Reply:
It looks _fine_. It looks like it was enlarged using nearest neighbor interpolation; but there's enough underlying resolution there that it doesn't matter much when we downsize it.

I think you need a plugin to actually download the original Figma image, which is crazy. It's really rare that we are actually pulling static image assets. When this does happen I try to hunt down the original from the designer or the source (for instance the artwork on Artsy).

Reply:
Are these image concerns a blocker @dzucconi? Or can we merge and follow-up?

Reply:
Oh sure; swapping images is totally a follow up thing.